src/gasPriceOracle/adapters/ethereum.ts

```diff
@@ -43,7 +43,11 @@ export async function eip1559Raw(
     provider.getBlock("pending"),
     (provider as providers.JsonRpcProvider).send("eth_maxPriorityFeePerGas", []),
   ]);
-  const maxPriorityFeePerGas = BigNumber.from(_maxPriorityFeePerGas);
+  let maxPriorityFeePerGas = BigNumber.from(_maxPriorityFeePerGas);
+  const flooredPriorityFeePerGas = parseUnits(process.env[`MIN_PRIORITY_FEE_PER_GAS_${chainId}`] || "0", 9);
```

Comment:
Question for reviewers: do you think its worth adding a default value for some chains here (e.g. "1": 0.5) to make it easier for API and relayer to be aligned? I kind of think it might be useful to keep relayer's floor value higher than the API and we can leave the API floor at 0 while we set the relayer's at 0.5 gwei

Reply:
no this could be set per-chain, so we could have a 0.5 gwei one for Ethereum